src/setup/node.rs

```diff
@@ -202,17 +244,16 @@ impl Drop for Node {
     }
 }
 
-pub fn build_stateful_path() -> io::Result<PathBuf> {
+fn get_stateful_node_path() -> io::Result<PathBuf> {
+    // TODO support multiple nodes
     let ziggurat_path = build_ripple_work_path()?;
-    Ok(ziggurat_path.join(NODE_STATE_DIR))
+    Ok(ziggurat_path.join(STATEFUL_NODES_DIR).join("1"))
 }
 
-pub fn build_stateful_builder(target: PathBuf) -> anyhow::Result<NodeBuilder> {
-    let source = build_stateful_path()?;
-    let testnet = TestNet::new()?;
-    Ok(Node::builder(Some(source), target)
+pub fn build_stateful_builder() -> anyhow::Result<NodeBuilder> {
```

Comment:
it's a mouthful; why not `stateful_node_builder`?

Reply:
or, WAY better, just make it a `default` method on `NodeBuilder`

Reply:
Yes, that sounds much better.